### PR TITLE
fix(deps): update most prod deps

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,57 +25,56 @@
...
     "grunt-rev": "0.1.0",
-    "grunt-sass": "1.0.0",
+    "grunt-sass": "1.1.0",
     "grunt-sri": "0.0.5",
     "grunt-text-replace": "0.4.0",
     "grunt-usemin": "3.1.1",
     "grunt-z-schema": "0.1.0",
-    "handlebars": "3.0.3",
+    "handlebars": "4.0.5",
     "helmet": "0.12.0",
```
Comment (vladikoff) [inline package.json]: This will need a different PR

File: package.json [hunk 2]
```diff
@@ -25,57 +25,56 @@
...
+    "lodash": "4.11.1",
     "marked": "0.3.5",
     "mkdirp": "0.5.1",
-    "morgan": "1.6.1",
-    "mozlog": "2.0.2",
+    "morgan": "1.7.0",
+    "mozlog": "2.0.3",
     "node-statsd": "0.1.1",
     "node-uap": "git://github.com/vladikoff/node-uap.git#9cdd16247",
     "openid": "1.0.0",
```
Comment (vladikoff) [inline package.json]: Same as https://github.com/mozilla/fxa-content-server/pull/3667/files#r59893853

File: package.json [hunk 3]
```diff
@@ -25,57 +25,56 @@
...
+    "grunt-requirejs": "git://github.com/vladikoff/grunt-requirejs.git#2ce7bac37f",
     "grunt-rev": "0.1.0",
-    "grunt-sass": "1.0.0",
+    "grunt-sass": "1.1.0",
     "grunt-sri": "0.0.5",
     "grunt-text-replace": "0.4.0",
     "grunt-usemin": "3.1.1",
     "grunt-z-schema": "0.1.0",
-    "handlebars": "3.0.3",
+    "handlebars": "4.0.5",
```
Comment (vladikoff) [inline package.json]: https://github.com/wycats/handlebars.js/blob/master/release-notes.md

**Comment by vladikoff** [comment]
@shane-tomlinson r?

File: package.json [hunk 4]
```diff
@@ -25,57 +25,56 @@
...
+    "grunt-contrib-clean": "1.0.0",
+    "grunt-contrib-concat": "1.0.0",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-cssmin": "1.0.1",
+    "grunt-contrib-htmlmin": "1.3.0",
+    "grunt-contrib-uglify": "1.0.1",
+    "grunt-githash": "0.1.3",
+    "grunt-marked": "0.1.3",
+    "grunt-po2json": "git://github.com/shane-tomlinson/grunt-po2json.git#2f415c8",
+    "grunt-requirejs": "git://github.com/vladikoff/grunt-requirejs.git#2ce7bac37f",
```
Comment (shane-tomlinson) [inline package.json]: What's this contain?

Comment (vladikoff) [inline package.json]: just peerDep update: https://github.com/vladikoff/grunt-requirejs/commits/peerDep

Comment (vladikoff) [inline package.json]: https://github.com/asciidisco/grunt-requirejs/pull/112

**Comment by vladikoff** [comment]
"still runs" :shipit:

**Pull request merged**